go/protocol/identify.go

```diff
@@ -1,4 +1,4 @@
-// Auto-generated by avdl-compiler v1.1.1 (https://github.com/keybase/node-avdl-compiler)
+// Auto-generated by avdl-compiler v1.2.0 (https://github.com/keybase/node-avdl-compiler)
```

Comment:
either run
````npm i``` in protocol to ensure being up to date, or mostly just notice a downgrade:
See this commit:
https://github.com/keybase/client/commit/7e63f276fea7d42c6d302c236cda1f7d166af061#diff-837099857ff4934ea30490256afc741c
we went from 1.2.0 to 1.1.1